peazip-sources/unit_gwrap.pas

```diff
@@ -2049,7 +2050,18 @@ procedure TForm_gwrap.FormShow(Sender: TObject);
 begin
 if pldesigned=true then exit;
 getdesk_env(desk_env,caption_build,delimiter);
+
+executable_path:=extractfilepath((paramstr(0))); //valorize application's paths
+if ((executable_path='') or (executable_path='..\')) then executable_path:=extractfilepath((paramstr(0)));
+if executable_path<>'' then
+   if executable_path[length(executable_path)]<>directoryseparator then executable_path:=executable_path+directoryseparator;
 setcurrentdir(executable_path);
+{$IFDEF DARWIN}
+   resource_path:=executable_path+'../Resources/';
+{$ELSE}
+   resource_path:=resource_path;

```

Comment:
resource_path:=executable_path+'res'+DirectorySeparator;

Reply:
Sorry, I have did "replace all from 'executable_path+'res'+DirectorySeparator' to 'resource_path'", and no found this has replaced either.

Reply:
No problem, it is fixed in the updated sources I published yesterday.